bedrock/firefox/templates/firefox/features/send-tabs.html

```diff
@@ -0,0 +1,151 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button with context %}
+
+{% add_lang_files "firefox/shared" "firefox/sync" %}
```

Comment:
There's one shared string ("It looks like you’re running an older version of Firefox.") - I guess I'm not sure if that warrants importing the Sync lang file or not.

@peiying2 or @flodolo  - Do you have an opinion here? There's one shared string between this template and `/firefox/features/sync.html`.